Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelCancelEditingTests.swift

```diff
@@ -9,28 +9,32 @@ final class SitterProfileViewModelCancelEditingTests: XCTestCase {
         // Given
         let viewModel = SitterProfileViewModel()
 
-        let newName = Sitter.Test.johnSmith.name
-        let newSurname = Sitter.Test.johnSmith.surname
-        let newPhone = Sitter.Test.johnSmith.phone
-        let newBio = Sitter.Test.johnSmith.bio
-        let newPricePerHour = Sitter.Test.johnSmith.pricePerHour
+        let originalName = viewModel.name
+        let originalSurname = viewModel.surname
+        let originalPhone = viewModel.phone
+        let originalBio = viewModel.bio
+        let originalPricePerHour = viewModel.pricePerHour
 
-        viewModel.name = newName
-        viewModel.surname = newSurname
-        viewModel.phone = newPhone
-        viewModel.bio = newBio
-        viewModel.pricePerHour = String(newPricePerHour)
+        viewModel.name = Sitter.Test.johnSmith.name
```

Comment:
```suggestion
       // When
        viewModel.name = Sitter.Test.johnSmith.name
```
Would it be better to place it here?

Reply:
Discussed it. Will stay as it is.